fbrp/examples/03_api/README.md

```diff
@@ -0,0 +1,23 @@
+# Example 03: API
+
+Here, we see an example of how to load a prebuilt docker image.
+
+The alephzero/api image we load is an AlephZero web-bridge.
+
+FBRP relays process stdout/stderr through AlephZero topics, allowing us to build pure-client side log monitoring.
```

Comment:
Would be nice to explain explicitly what part the web-bridge plays in this.
I'm assuming that the web-bridge is a server-like process that hosts topics, which `alice` and `bob` both connects to.